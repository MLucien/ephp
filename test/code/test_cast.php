<?php

var_dump([
    (int)26.7,
    (int)"40",
    (int)45,
    (float)"56.76",
    (float)56.76,
    (float)55,
    (string)100,
    (string)25.52,
    (string)"world",
    (bool)1,
    (bool)"",
    (bool)25.52,
    (array)"hello",
    (array)100,
    (array)45.54
]);
