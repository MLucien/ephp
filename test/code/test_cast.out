array(15) {
  [0]=>
  int(26)
  [1]=>
  int(40)
  [2]=>
  int(45)
  [3]=>
  float(56.76)
  [4]=>
  float(56.76)
  [5]=>
  float(55)
  [6]=>
  string(3) "100"
  [7]=>
  string(5) "25.52"
  [8]=>
  string(5) "world"
  [9]=>
  bool(true)
  [10]=>
  bool(false)
  [11]=>
  bool(true)
  [12]=>
  array(1) {
    [0]=>
    string(5) "hello"
  }
  [13]=>
  array(1) {
    [0]=>
    int(100)
  }
  [14]=>
  array(1) {
    [0]=>
    float(45.54)
  }
}
